maggy/src/services/executor.py

```
@@ -0,0 +1,253 @@
+            "task_id": task_id,
+            "task_title": task.title,
+            "mode": mode,
+            "working_dir": wd,
+            "status": "running",
+            "started_at": datetime.now(timezone.utc).isoformat(),
+            "output": "",
+        }
+        self._sessions[session_id] = session
+
+        # Run in background
+        asyncio.create_task(self._run(session_id, task, wd, mode))
```

Ruff (0.15.10)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)